### PR TITLE
every single form has extra white space on the right on mobile

### PR DESCRIPTION
**Pull request by weyermannx**
🎨 
* added some helpers to I can make mobile-only changes (for payment pages)
https://github.com/amaabca/membership/pull/717 - payment changes
* changed summary page image to be left-aligned on mobile

### CODE REVIEW

**Comment by weyermannx** [comment]
:wave:

File: app/assets/stylesheets/layout/helper-classes.scss [hunk 1]
```diff
@@ -35,6 +35,11 @@
 
 .right{
   float: right !important;
+  &--mobile{
+    @include breakpoint(small only){
+      float: right !important;
```
Comment (sineade) [inline app/assets/stylesheets/layout/helper-classes.scss]: do these need to have the !important?

Comment (weyermannx) [inline app/assets/stylesheets/layout/helper-classes.scss]: not sure...  let me check

**Comment by sineade** [comment]
:art: :art: :art:

**Pull request merged**